text/0232-maturity-of-otel.md

```diff
@@ -0,0 +1,151 @@
+# Framework for defining the maturity of OpenTelemetry and its SIGs
+
+On 08 Mar 2023, the OpenTelemetry GC and TC held an OpenTelemetry Leadership summit, discussing various topics. One of the themes we discussed was establishing standard rules for describing the maturity of the OpenTelemetry project. This OTEP summarizes what was discussed there and is intended to have the wider community provide feedback.
+
+This OTEP builds on what was previously communicated by the project, especially the [Versioning and stability for OpenTelemetry clients](https://opentelemetry.io/docs/reference/specification/versioning-and-stability).
+
+The Collector's [stability levels](https://github.com/open-telemetry/opentelemetry-collector#stability-levels) inspired the maturity levels.
+
+## Motivation
+
+Quite often, the community is faced with the question of the quality and maturity expectations of its diverse set of components. This OTEP aims to bring clarity by establishing a framework to communicate the project's maturity. As the OpenTelemetry project comprises a multitude of SIGs, and each SIG has several components of varying quality, having this framework will help set the right expectations for OpenTelemetry users.
+
+Only once we reach the "stable" level at the project level will we be ready to consider graduating OpenTelemetry at the CNCF, with all of the Tier-1 SIGs being in the context of the graduation, including their Tier-1 Components. This OTEP provides a framework for describing project maturity as a whole.
+
+## Glossary
+
+* *SIG:* an official SIG within the OpenTelemetry project, like the Collector or Java. Some WGs are temporarily accepted, given they should be SIGs instead of WGs, like the Semantic Conventions WG.
```

Comment:
> Some WGs are temporarily accepted, given they should be SIGs instead of WGs, like the Semantic Conventions WG.

Not sure what this sentence means.

Reply:
Sorry, I should have been clearer, and I accept suggestions. The wording is to account for the fact that we have WGs that should really be SIGs, as the "Semantic Conventions WG". So, when we say "SIG" in this doc, we mean actual SIGs plus those WGs that should be SIGs :-)

Even more context: during a TC (or GC?) call, someone asked what's the difference between a SIG and a WG. The consensus seems to be that WGs are geared towards and end-goal, and once that's reached, the WG is dissolved. SIGs are long-term, they don't have an end.

Reply:
I'm marking this as resolved, let me know if that's not the case.

Reply:
Actually, I'm reopening as I changed the text and would appreciate another review, verifying it's clearer now.